recipes/freecad/meta.yaml

```diff
@@ -0,0 +1,100 @@
+{% set name = "freecad" %}
+{% set version = "0.18b0" %}
+{% set sha256 = "1462df34caa67791bb7512f6c464f16a15d49f0d9f777996cb4e7b8174c57191" %}
+{% set commit = "e49a5af3ae2f976017f1156bdc113594f81d3fb5" %}
+
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}_{{ version }}.tar.gz
+    url: https://github.com/FreeCAD/FreeCAD/archive/{{ commit }}.tar.gz
+    sha256: {{ sha256 }}
+    patches:
+        - osx.patch
+        - smesh.patch
+        - win_unicode.patch
+
+build:
+    number: 0
+    skip: True  # [py < 36]
+
+requirements:
+    build:
+        - {{ compiler("cxx") }}
+        - vc 14  # [win]
+        - ninja
+        - cmake
+    host:
+        - nomkl  # [unix]
+        - boost-cpp 1.66.0
+        - python
+        - qt 5.6.*
+        - occt 7.2.*
+        - xerces-c 3.2.*
+        - zlib 1.2.*
+        - vtk 8.1.1
+        - swig
+        - eigen
+        - pybind11
+        - matplotlib
+        - coin3d
+        - pivy
+        - pyside2
+        - smesh
+        - netgen  # [not win32]
+        - doxygen
+        - libglu  # [linux]
+        - mesalib  # [linux]
+        - gcc  # [linux]
+        - hdf5 1.10.1
+    run:
+        - {{ pin_compatible("qt") }}
+        - {{ pin_compatible("pyside2") }}
+        - {{ pin_compatible("occt") }}
+        - {{ pin_compatible("vtk") }}
+        - {{ pin_compatible("xerces-c") }}
+        - {{ pin_compatible("zlib") }}
+        - {{ pin_compatible("boost-cpp") }}
+        - {{ pin_compatible("hdf5") }}
```

Comment:
Remove the `pin_compatible`s from `zlib, xerces-c, boost-cpp, hdf5, qt`. For vtk, you need `{{ pin_compatible("vtk", max_pin='x.x.x') }}` so that it's pinned to the patchlevel